src/web/app/pages-logs/logs-page.component.ts

```diff
@@ -129,11 +141,50 @@ export class LogsPageComponent implements OnInit {
               (e: ErrorMessageOutput) => this.statusMessageService.showErrorToast(e.error.message));
   }
 
+  searchForLogsHistogramView(localDateTime: Observable<number>[]): void {
+    forkJoin(localDateTime)
```

Comment:
Note that this will be affected by #11286

Reply:
ok! I will take note of this.